### ISSUE TITLE
Default text - should be newest gem

### ISSUE THREAD

**Issue by jdsundberg** [quirkey/gembox#1]
It would be far more valuable to see what is the newest gem -- vs saying the oldest gem.

Generally - the new stuff is what you are working with. The oldest -- will never change.

**Comment by benatkin** [comment]
I agree. I [implemented this](http://github.com/benatkin/gembox/commit/f1be65c323183ad39c69a8b40bbb825e9e95236d) in [my fork of gembox](http://github.com/benatkin/gembox/tree/master).
